UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.UnitTests/Extensions/DateTimeExtensionsTests.cs

```diff
@@ -48,5 +48,12 @@ public void CheckIsValidDateWithValidRfc1123FormatLeapYear_ThenReturnTrue()
             bool isValidDate = DateTimeExtensions.IsValidRfc1123Format("Tue, 29 Feb 2000 12:16:24 GMT", out DateTime fakeFolderDateTime);
             Assert.IsTrue(isValidDate);
         }
+
+        [Test]
+        public void CheckIsValidWeek_ThenReturnTrue()
```

Comment:
This test name is confusing and also I'm unsure we need a method to turn days into weeks,

Reply:
Removed it as not need now.